CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ ___
 
 ___
 
+## Version 2.2.0 (Under development)
+
+### App Center
+
+* **[Feature]** Add `AppCenter.isRunningInAppCenterTestCloud` to provide method to check if the application is running in Test Cloud.
```

Comment:
```suggestion
* **[Feature]** Add `isRunningInAppCenterTestCloud` in `MSAppCenter` to provide method to check if the application is running in Test Cloud.
```

Reply:
There is a bad merge in the file. It always happens and there is one more in the file. Would you please use one `## Version 2.2.0 (Under development)`?